zio-config/src/main/scala/zio/config/Config.scala

```diff
@@ -37,4 +42,18 @@ object Config {
 
   final case class Or[A, B](left: Config[A], right: Config[B]) extends Config[Either[A, B]]
 
+  def sequence[A](configList: List[Config[A]]): Config[List[A]] =
```

Comment:
Be sure to have `collectAll` alias like ZIO here.

Reply:
@jdegoes This is done.